basic/okta_sdk/okta_sdk.tf

```
@@ -12,13 +12,13 @@ resource "sym_integration" "okta" {
 }
 
 # An AWS Secrets Manager Secret to hold your Okta API Key. Set the value with:
-# aws secretsmanager put-secret-value --secret-id "main/okta-api-key" --secret-string "YOUR-OKTA-API-KEY"
+# aws secretsmanager put-secret-value --secret-id "${local.environment_name}/okta-api-key" --secret-string "YOUR-OKTA-API-KEY"
 resource "aws_secretsmanager_secret" "okta_api_key" {
-  name        = "main/okta-api-key"
+  name        = "${local.environment_name}/okta-api-key"
```

```suggestion
  name        = "sym/${local.environment_name}/okta-api-key"
```

Since we started prefixing all the secrets in our generated code with `sym/`?